source/routes/read/month.js

```diff
@@ -0,0 +1,96 @@
+const PouchDB = require('pouchdb');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-sp21-group13/cse110-sp21-group13/pullRequest?prid=7423478)
```suggestion
const PouchDB = require("pouchdb");
```